test/module/irohad/torii/processor/transaction_processor_test.cpp

```diff
@@ -126,11 +138,11 @@ TEST_F(TransactionProcessorTest, TransactionProcessorOnProposalTest) {
       }));
 
   EXPECT_CALL(*mp, propagateTransactionImpl(_)).Times(0);
-  EXPECT_CALL(*pcs, propagate_transaction(_)).Times(txs.size());
+  EXPECT_CALL(*pcs, propagate_batch(_)).Times(txs.size());
```

Comment:
maybe rename to `propagateBatch`?

Reply:
All methods in PCS are called in this way, I think, it's too much refactoring for this PR :(